controllers/topic.controllers.js

```diff
@@ -0,0 +1,9 @@
+const { selectTopics } = require("../models/topics.models.js");
+
+exports.getTopic = (req, res, next) => {
+  selectTopics()
+    .then((topics) => {
+      res.status(200).send({ 
+      topics: topics });
+    })
```

Comment:
You've invoked this function with next but no catch block. This particular function does not require it as there can be no 404 or 400 when requesting an array of topics.